scifight/migrations/0001_initial.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
```

Comment:
Why didn't you created a proper migration file, but just updated the initial migration?